Dockerfile

```diff
@@ -1,24 +1,12 @@
 ARG  NODE_VERSION
 FROM node:${NODE_VERSION}
 
-RUN apk --no-cache add git
+WORKDIR /app
+RUN corepack enable
 
-# Create app directory
-RUN mkdir -p /usr/src/app/.nuxt
-WORKDIR /usr/src/app
-
-# Install app dependencies
-COPY package.json yarn.lock /usr/src/app/
-RUN yarn install
-RUN yarn add -D serve
-COPY . /usr/src/app
+COPY . .
 RUN yarn install
```

Comment:
Should keep the two steps:
1. Add  package.json yarn.lock 
2. yarn install
3. Add all
4. build

It allows to cache into docker the yarn install steps.

Reply:
Yep I agree we need to take benefit deps caching. But I'm not very confortable with this double `yarn install` I wish I can find a way to re-use node_module produce in the first time ...

Reply:
Just keep it at it was before.

Reply:
I make the changes.

Reply:
What do you mean ? The whole Dockerfile ?

Reply:
We can discuss about it if you want. To me the only unclear part is the double `yarn install`.

Reply:
Yes. After the first `yarn install` there is no `node_modules`.

Reply:
Yes and it's normal behavior from Docker as I understand.
It looks like we have several options that I will detail later